thumbor/engines/gif.py

```diff
@@ -118,8 +118,7 @@ def read(self, extension=None, quality=None):
 
         # Make sure gifsicle produced a valid gif.
         try:
-            with Image.open(BytesIO(self.buffer)) as image:
-                image.verify()
+            Image.open(BytesIO(self.buffer)).verify()
```

Comment:
Ah, here it is. However, `verify` releases the pointer: https://github.com/python-pillow/Pillow/blob/ba5f9b88da66b80aa7311ade601a5caf3d05401b/PIL/ImageFile.py#L118
so when context manager was trying to close it - it throws an error. I'll figure out a better way.